docs/concepts/dns.md

```diff
@@ -0,0 +1,79 @@
+# DNS
+
+Lokomotive relies on DNS records for cluster bootstrap as well as routine operation. When choosing
+one of the [supported DNS providers](#supported-providers), Lokomotive can provision the necessary
+DNS records on your behalf.
+
+## Supported providers
+
+- [AWS Route 53](https://aws.amazon.com/route53/) - supported with AWS and Packet clusters
+- [Cloudflare](https://www.cloudflare.com/dns/) - supported with Packet clusters
+- Manual - supported with Packet clusters
+
+>NOTE: The *manual* DNS provider is a special provider. When using this option, no DNS records are
+>provisioned automatically. Instead, the user is prompted to configure the necessary DNS records on
+>their own.
+
+The records on which Lokomotive relies are mostly DNS **A records**. They are constructed based on
+the DNS zone provided by the user and the configured cluster name in the following format:
+
+    <record>.<cluster_name>.<zone>
+
+The exact set of records created varies slightly based on the platform on top of which Lokomotive
+is deployed. The following sections provide a per-platform explanation.
+
+## AKS
+
+TODO
```

Comment:
```suggestion
Lokomotive deployment using AKS does not require DNS configuration, so it's not creating any records, as AKS is a managed platform, so Azure is doing it on user's behalf.
```

Reply:
Thanks. I've included this info using a different phrasing.